manifests/prometheus/dashboards.d/cell-memory.json

```diff
@@ -0,0 +1,701 @@
+{
+  "dashboard": {
+    "annotations": {
+      "list": [
+        {
+          "builtIn": 1,
+          "datasource": "-- Grafana --",
+          "enable": true,
+          "hide": true,
+          "iconColor": "rgba(0, 211, 255, 1)",
+          "name": "Annotations & Alerts",
+          "type": "dashboard"
+        }
+      ]
+    },
+    "editable": true,
+    "gnetId": null,
+    "graphTooltip": 1,
+    "id": null,
+    "iteration": 1666363845398,
+    "links": [],
+    "panels": [
+      {
+        "cards": {
+          "cardPadding": null,
+          "cardRound": null
+        },
+        "color": {
+          "cardColor": "#b4ff00",
+          "colorScale": "sqrt",
+          "colorScheme": "interpolatePlasma",
+          "exponent": 0.5,
+          "mode": "spectrum"
+        },
+        "dataFormat": "timeseries",
+        "datasource": null,
+        "description": "Amount of available memory advertised by rep divided by the actual amount of available memory on cell.\n\nA value much greater than 100% suggests a cell that might be in danger of running out of actual memory and having app memory go into swap or even have apps unexpectedly crash due to machine OOM.\n\nA value much less than 100% suggests a cell that is probably stopping accepting new app instances when it still has plenty of actual memory left. Suboptimal but safe.",
```

Comment:
```suggestion
        "description": "Available memory advertised by the Reps divided by the actual available memory on the cell.\n\nA value much greater than 100% suggests a cell that might be in danger of running out of actual memory, resulting in app memory going into swap or apps unexpectedly crashing due to machine OOM.\n\nA value much lower than 100% suggests a cell that is not accepting new app instances when it still has plenty of actual memory left. It is suboptimal but safe.",
```

Reply:
> the Reps

I would probably make it `the Rep` if anything because I wanted to make it clear that this is a value calculated per-rep.

> is not accepting new app instances

This was deliberately in the future tense because it's more that it's projecting that it _will_ stop accepting new apps early - it may still be accepting apps.

Reply:
ah good, I was wondering about that plural.

For the second one, we could do:

> ...suggests a cell that might stop accepting new app instances